sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1247,7 +1245,7 @@ class MapboxMap internal constructor(
 
   internal fun setCameraAnimationPlugin(cameraAnimationsPlugin: CameraAnimationsPlugin?) {
     cameraAnimationsPlugin?.let {
-      if (it.javaClass.canonicalName == PLUGIN_CAMERA_ANIMATIONS_CLASS_NAME)
+      if (it is CameraAnimationsPluginImpl)
```

Comment:
same as https://github.com/mapbox/mapbox-maps-android/pull/519/files#r677527136

Reply:
Will resolve it here, let's not duplicate discussions 😉